doc_template/examples_root/examples/nb/mouse_connectivity.ipynb

```diff
@@ -2,10 +2,19 @@
  "cells": [
```

Comment:
<p>Any idea why we lost 2 experiments?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/AllenSDK/pull/2617/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
I do not.